roles.json

```
@@ -207,29 +207,110 @@
+        "role": "DevOps and Operations Specialist",
+        "people": [
+            "E. Madison Bray",
+            "Pey Lian Lim",
+            "Brigitta Sip\u0151cz"
+        ]
+        },
+        {
+        "role": "CI Infrastructure",
         "people": [
             "Simon Conseil",
             "Pey Lian Lim",
             "Thomas Robitaille",
             "Brigitta Sip\u0151cz"
-        ],
-        "role-head": "Documentation infrastructure maintainer",
-        "responsibilities": {
-            "description": "Maintain the <a href='http://docs.astropy.org/en/stable/index.html'>Astropy documentation</a> website, including:",
-            "details": [
-                "Managing the Sphinx infrastructure",
-                "Implementing changes and improvements to the documentation website",
-                "Overseeing content (although primary responsibility for content lies with subpackage maintainers)"
-            ]
+        ]
+        },
+        {
+        "role": "Documentation Infrastructure",
+        "people": [
+            "Simon Conseil",
+            "Pey Lian Lim",
+            "Thomas Robitaille",
+            "Brigitta Sip\u0151cz"
+        ]
+        },
+        {
+        "role": "Metrics",
+        "people": ["Unfilled"]
+        },
+        {
+        "role": "Website and Data",
+        "people": [
+            "Adrian Price-Whelan",
+            "Erik Tollerud"
+        ]
+        }
+    ],
+    "responsibilities": [
+        {
+        "subrole-head": "DevOps and Operations Specialist",
+        "description": "Ensure the smooth running of the project:",
+        "details": [
+            "Perform initial triaging of issues and pull requests, including moving between repositories in the project.",
+            "Perform first responder duties across the project, including guiding first-time contributors.",
+            "Monitor the health of the project in terms of development and operations; if necessary, also alert the relevant project members of any issues and/or formulate responses.",
+            "Monitor the CI statuses and promptly report any CI breakage to the relevant project members.",
+            "Merge non-controversial pull requests in the project.",
+            "Liaison with other projects that share similar infrastructure."
+        ]
+        },
+        {
+        "subrole-head": "CI Infrastructure",
+        "description": "Lead development and maintenance of the continuous integration (CI) testing infrastructure for Astropy, including:",
+        "details": [
+            "Set up and maintain continuous integration services.",
+            "Managing issues/pull request for the astropy core package regarding testing infrastructure.",
+            "Managing issues/pull requests in the repositories containing the testing plugins, and determining when new plugins are required.",
+            "Maintaining the 'metapackage' with the testing machinery (pytest-astropy at the time of this writing).",
```

```suggestion
            "Maintaining the 'metapackage' with the testing machinery (e.g. pytest-astropy).",
```
While, strictly speaking there is a git commit that specifies "at the time of this writing", on the website you don't see that. So, I suggest are more generic wording even at the risk of that being imprecise at some point in the future.

---

Accepted your change. Thanks!